spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -217,35 +218,22 @@
     describe 'IPP enabled' do
       describe 'DAV not enabled' do
         let!(:pending_enrollments) do
-          [
-            create(
-              :in_person_enrollment, :pending, :with_notification_phone_configuration,
-              selected_location_details: { name: 'BALTIMORE' },
-              issuer: 'http://localhost:3000'
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'FRIENDSHIP' }
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'WASHINGTON' }
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'ARLINGTON' }
-            ),
-            create(
-              :in_person_enrollment, :pending,
-              selected_location_details: { name: 'DEANWOOD' }
-            ),
-          ]
+          locations = ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD']
+          build_list(
+            :in_person_enrollment, 5, :pending,
+            :with_notification_phone_configuration
+          ) do |record, i|
+            record.issuer = 'http://localhost:3000'
+            record.selected_location_details = { name: locations[i] }
+            record.save!
+          end
```

Comment:
maybe easier to do a `map`?
```suggestion
          ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD'].map do |name|
            create(
              :in_person_enrollment,
              :pending, 
              issuer: 'http://localhost:3000',
              selected_location_details: { name: name },
            )
```